packages/siwx/src/verifiers/SolanaVerifier.ts

```diff
@@ -0,0 +1,22 @@
+import { SIWXVerifier } from '../core/SIWXVerifier.js'
+import type { SIWXSession } from '@reown/appkit-core'
+import nacl from 'tweetnacl'
+import bs58 from 'bs58'
+
+export class SolanaVerifier extends SIWXVerifier {
+  public readonly chainNamespace = 'solana'
```

Comment:
If possible we could import ConstantsUtil from `common` package and do this like
```suggestion
  public readonly chainNamespace = ConstantsUtil.CHAIN.SOLANA
```